src/lib/containers/entity_finder/details/configurations/EntityHeaderListDetails.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { EntityHeader, ProjectHeader } from '../../../../utils/synapseTypes'
+import { DetailsView } from '../view/DetailsView'
+import { EntityDetailsListSharedProps } from '../EntityDetailsList'
+
+type EntityHeaderListDetailsProps = EntityDetailsListSharedProps & {
+  entityHeaders: (EntityHeader | ProjectHeader)[]
+}
+
+export const EntityHeaderListDetails: React.FunctionComponent<EntityHeaderListDetailsProps> = ({
```

Comment:
why does this wrapper exist (why not just use a DetailsView directly in this case)?